bin/sandbox

```diff
@@ -49,13 +49,8 @@ else
   GENERATOR_SKIP_SPECS_ARGUMENT=''
 fi
 
-# Stay away from the bundler env of the containing extension.
-function unbundled {
-  ruby -rbundler -e'b = proc {system *ARGV}; Bundler.respond_to?(:with_unbundled_env) ? Bundler.with_unbundled_env(&b) : Bundler.with_clean_env(&b)' -- $@
-}
```

Comment:
`Bundler.with_unbundled_env` / `Bundler.with_clean_env` provides us with a completely Bundler-free environment. What we want instead is for the sandbox app to recognize its own bundle.

See https://makandracards.com/makandra/15649-how-to-discard-a-surrounding-bundler-environment for an explanation of the scenario we have here: a directory and a subdirectory both with their bundles.